### DISCUSSION TITLE
persistent volume not being read/used

### DISCUSSION THREAD

**Discussion by wysegy66** [Keyfactor/ejbca-ce#677] [Q&A]
I set up a test EJBCA-CE instance using docker and a persistent volume on my local machine (Mac).
I created some CA profiles and crypto tokens ...... I was able to shut down and come back later and still see all my data.
Now when I go back and run it, all my previous setup is missing and it looks like a new default configuration. 

This is what I'm using the start the instance:

`docker run -it --rm -p 80:8080 -p 443:8443 -h TESTMACHINE -e TLS_SETUP_ENABLED="simple" -e DATABASE_JDBC_URL="jdbc:h2:/mnt/persistent/ejbcadb;DB_CLOSE_DELAY=-1" -v $TEST_DIR/ejbca-data:/mnt/persistent keyfactor/ejbca-ce`

where $TEST_DIR is a local directory on my Mac

E.g, I have a backup of the good old data and for example there is a ejbcadb.mv.db file dated Sept 20 and 299kb in size
When I copy the old data into $TEST_DIR/ejbca-data and start the container it's like the old data does not get read and I'm at a default config.  When I exit the ejbcadb.mv.db file is only 133kb in size.

Any ideas?

**Comment by danieldownload** [comment]
Make sure you that the owner of the volume is correct. On Redhat for example it must be 1001:1001
